main.tex

```diff
@@ -174,6 +163,8 @@ \section*{Introduction}
 %
 We use genotype and haplotype data derived from whole-genome Illumina sequencing of 765 individual mosquitoes collected from natural populations in 8 African countries to survey genetic diversity and study the evolutionary and demographic history of insecticide resistance at the \emph{Vgsc} locus.
 %
+Initial Ag1000g findings are confirmed then analyses extended to investigate the age, movement and recombination of insecticide resistance conferring haplotypes.
+%
 Our results reveal an unexpected diversity of molecular mechanisms of resistance, and shed new light on the evolutionary processes underlying the rapid increase in the prevalence of resistance across multiple mosquito populations.
 %%
 
```

Comment:
Nit, but we are repeating the abstract here. Instead, how about:

"Our aim is to provide a comprehensive account of genetic variation at the \emph{Vgsc} locus and its implications for the management of pyrethroid resistance in natural vector populations."